draft-ietf-rtcweb-jsep.xml

```diff
@@ -1360,10 +1360,12 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
           things, it sets up local resources for receiving and decoding
           media. In order to successfully handle scenarios where the
           application wants to offer to change from one media format to
-          a different, incompatible format, the PeerConnection must be
-          able to simultaneously support use of both the current and
-          pending local descriptions (e.g. support codecs that exist in
-          both descriptions) until a final answer is received, at which
+          a different, incompatible format, while in the have-local-offer
```

Comment:
This sentence is complicated. I would suggest splitting this into two sentences, e.g.

> In order to successfully handle scenarios where the application wants to offer to change from one media format to a different, incompatible format, the PeerConnection must be able to simultaneously support use of both the current and pending local descriptions (e.g. support the codecs that exist in either description). This dual processing begins when the PeerConnection enters the have-local-offer state, and continues until setRemoteDescription is called with either a final answer, at which point the PeerConnection can fully adopt the pending local description, or a rollback, which results in a revert to the current local description.